public/assets/cms/public.js

```diff
@@ -11,7 +11,7 @@
h:st,sort:ot.sort,splice:ot.splice},ft.extend=ft.fn.extend=function(){var t,e,o,i,r,n,s=arguments[0]||{},a=1,p=arguments.length,l=!1;for("boolean"==typeof s&&(l=s,s=arguments[a]||{},a++),"object"==typeof s||ft.isFunction(s)||(s={}),a===p&&(s=this,a--);p>a;a++)if(null!=(r=arguments[a]))for(i in r)t=s[i],o=r[i],s!==o&&(l&&o&&(ft.isPlainObject(o)||(e=ft.isArray(o)))?(e?(e=!1,n=t&&ft.isArray(t)?t:[]):n=t&&ft.isPlainObject(t)?t:{},s[i]=ft.extend(l,n,o)):void 0!==o&&(s[i]=o));return s},ft.extend({expando:"jQuery"+(ct+Math.random()).replace(/\D/g,""),isReady:!0,error:function(t){throw new Error(t)},noop:function(){},isFunction:function(t){return"function"===ft.type(t)},isArray:Array.isArray||function(t){return"array"===ft.type(t)},isWindow:function(t){return null!=t&&t==t.window},isNumeric:function(t){var e=t&&t.toString();return!ft.isArray(t)&&e-parseFloat(e)+1>=0},isEmptyObject:function(t){var e;for(e in t)return!1;return!0},isPlainObject:function(t){var e;if(!t||"object"!==ft.type(t)||t.nodeType||ft.isWindow(t))return!1;try{if(t.constructor&&!ut.call(t,"constructor")&&!ut.call(t.constructor.prototype,"isPrototypeOf"))return!1}catch(o){return!1}if(!ht.ownFirst)for(e in t)return ut.call(t,e);for(e in t);return void 0===e||ut.call(t,e)},type:function(t){return null==t?t+"":"object"==typeof t||"function"==typeof t?pt[lt.call(t)]||"object":typeof t},globalEval:function(e){e&&ft.trim(e)&&(t.execScript||function(e){t.eval.call(t,e)})(e)},camelCase:function(t){return t.replace(yt,"ms-").replace(gt,mt)},nodeName:function(t,e){return t.nodeName&&t.nodeName.toLowerCase()===e.toLowerCase()},each:function(t,e){var i,r=0;if(o(t))for(i=t.length;i>r&&e.call(t[r],r,t[r])!==!1;r++);else for(r in t)if(e.call(t[r],r,t[r])===!1)break;return t},trim:function(t){return null==t?"":(t+"").replace(dt,"")},makeArray:function(t,e){var i=e||[];return null!=t&&(o(Object(t))?ft.merge(i,"string"==typeof t?[t]:t):st.call(i,t)),i},inArray:function(t,e,o){var i;if(e){if(at)return at.call(e,t,o);for(i=e.length,o=o?0>o?Math.max(0,i+o):o:0;i>o;o++)if(o in e&&e[o]===t)return o}return-1},merge:function(t,e){for(var o=+e.length,i=0,r=t.length;o>i;)t[r++]=e[i++];if(o!==o)for(;void 0!==e[i];)t[r++]=e[i++];return t.length=r,t},grep:function(t,e,o){for(var i,r=[],n=0,s=t.length,a=!o;s>n;n++)i=!e(t[n],n),i!==a&&r.push(t[n]);return r},map:function(t,e,i){var r,n,s=0,a=[];if(o(t))for(r=t.length;r>s;s++)n=e(t[s],s,i),null!=n&&a.push(n);else for(s in t)n=e(t[s],s,i),null!=n&&a.push(n);return nt.apply([],a)},guid:1,proxy:function(t,e){var o,i,r;return"string"==typeof e&&(r=t[e],e=t,t=r),ft.isFunction(t)?(o=rt.call(arguments,2),i=function(){return t.apply(e||this,o.concat(rt.call(arguments)))},i.guid=t.guid=t.guid||ft.guid++,i):void 0},now:function(){return+new Date},support:ht}),"function"==typeof Symbol&&(ft.fn[Symbol.iterator]=ot[Symbol.iterator]),ft.each("Boolean Number String Function Array Date RegExp Object Error Symbol".split(" "),function(t,e){pt["[object "+e+"]"]=e.toLowerCase()});var vt=/*!
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Confusing use of '!'.
* Expected a conditional expression and instead saw an assignment.
* Too many errors. (13% scanned).

<!-- Comment by [SideCI](https://sideci.com) -->